modules/sfp_peegeepee.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:        sfp_peegeepee
+# Purpose:     SpiderFoot plug-in for looking up e-mail addresses and domains
+#              on PeeGeePee.com.
+#
+# Author:      <bcoles@gmail.com>
+#
+# Created:     2019-07-07
+# Copyright:   (c) bcoles
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import re
+import time
+import urllib
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+
+
+class sfp_peegeepee(SpiderFootPlugin):
+    """PeeGeePee:Footprint,Investigate,Passive:Public Registries::Look up e-mail addresses and domains on PeeGeePee.com."""
+
+    results = dict()
+
+    # Default options
+    opts = {
+        'timeout': 15,
+        'fetch_keys': False
+    }
+
+    # Option descriptions
+    optdescs = {
+        'timeout': 'Query timeout, in seconds.',
+        'fetch_keys': 'Retrieve PGP keys for each match.'
+    }
+
+    currentEventSrc = None
+    results = dict()
+
+    def setup(self, sfc, userOpts=dict()):
+        self.sf = sfc
+        self.__dataSource__ = 'PeeGeePee'
```

Comment:
FYI this doesn't need to be set, as by default the module name will be used. It only needs to be set when the module name isn't an adequate description of the data source.

Reply:
Removed